kernel/common/ezautolinkoperator.php

```diff
@@ -10,6 +10,13 @@
 
 class eZAutoLinkOperator
 {
+    static $max = 72;
+    
+    static $isLinkWwwUrl = false;
```

Comment:
This variable name is not so good, as it suggests by it's name (to us at any level) that the link in question is a www. style link but in truth the variable is only a boolean indicating that the operator use link www. type text. For these reasons we think this variable name should be changed. `$linkWwwUrls` (plural) is in our opinion a much better name. We request you change this variable name.